pom.xml

```diff
@@ -64,7 +64,7 @@
         <hamcrest-all.version>1.3</hamcrest-all.version>
         <junit.version>4.12</junit.version>
         <maven-plugin-testing-harness.version>3.3.0</maven-plugin-testing-harness.version>
-        <mockito-core.version>2.8.47</mockito-core.version>
+        <mockito-core.version>3.0.0</mockito-core.version>
```

Comment:
❓Do we even use mockito in this repo? If not, this dependency should be removed. If so, we should manually test that this major version bump doesn't break anything.

Reply:
Unless this is generated somewhere, I think it might have been accidentally left in when I copied stuff from the Discovery-API.

Reply:
I will try removing it.

Reply:
And I'll also check if there are any other unused dependencies in this repo.

Reply:
https://github.com/connexta/ion-ingest-api/pull/38